bot.py

```
@@ -2,544 +2,37 @@
-# Configure logging
-logging.basicConfig(
-    level=logging.INFO,
-    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
-    handlers=[
-        logging.FileHandler(log_filename),
-        logging.StreamHandler()
-    ]
-)
-logger = logging.getLogger('discord_bot')
+from logging_config import logger # Import the logger from the new module
+from rate_limiter import check_rate_limit, update_rate_limit_config # Import rate limiting functions
```

Ruff (0.11.9)

`rate_limiter.check_rate_limit` imported but unused

Remove unused import

(F401)